extensions/schema-compare/src/schemaCompareMainWindow.ts

```diff
@@ -101,7 +101,8 @@ export class SchemaCompareMainWindow {
 		this.editor.registerContent(async view => {
 			this.differencesTable = view.modelBuilder.table().withProperties({
 				data: [],
-				height: 300
+				height: 300,
+				title: localize('schemaCompare.differencesTableTitle', "Comparison between Source and Target")
```

Comment:
No need for an `aria-label`?  Be sure a screen-reader will announce it, too.

Reply:
yup NVDA announces it :)